docs/developers/references/traits/autoscale.md

```diff
@@ -0,0 +1,100 @@
+# Automatically scale workload by cron or resource utilization
```

Comment:
Put this to *docs/developers/set-autoscaling.md* instead.

Reply:
Sure.